src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -8,8 +8,12 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;netcoreapp3.0-Windows_NT;netcoreapp3.0-Unix</TargetFrameworks>
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);nullable</NoWarn>
   </PropertyGroup>
-  <ItemGroup>
+  <PropertyGroup>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsBrowser)' == 'true'">SR.SystemDrawingCommon_PlatformNotSupported</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
I thought `System.Drawing.Common` was going to throw just on the APIs that used `libgdiplus`?

Reply:
Most (if not all) types in `System.Drawing.Common` call into `libgdiplus` so we think it should just throw PNSE.

Reply:
I think you're right.